packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.tsx

```diff
@@ -7,5 +7,7 @@ export { IDetailsHeaderProps };
 
 export const DetailsHeader = styled<IDetailsHeaderProps, IDetailsHeaderStyleProps, IDetailsHeaderStyles>(
   DetailsHeaderBase,
-  getStyles
+  getStyles,

```

Comment:
Rename to DetailsHeader.ts (same thing for the other files like this)

Reply:
Done renaming